L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitivesAnalyzer_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Is this code only for testing? If yes, you should move it to `/test`. If not, `The _cfi file should be left to be generated automatically with the fillDescriptions()` (see https://cms-sw.github.io/cms_coding_rules.html and https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#Converting_an_Existing_CFI_file)

Reply:
Moved into the cfg.py file